tests/OpenPlatformTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use DDB\OpenPlatform\OpenPlatform;
+use DDB\OpenPlatform\Request\GenericRequest;
+use DDB\OpenPlatform\Request\SearchRequest;
+use DDB\OpenPlatform\Response\Response;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+
+// phpcs:ignore PSR1.Classes.ClassDeclaration.MissingNamespace
```

Comment:
Is there any reason why we would not want to add a namespace here?

See also other test cases for the same comment.